src/movie/List.js

```diff
@@ -111,18 +90,18 @@ export class MovieList extends Component {
             id="movieSortDropdown"
             onChange={this.handleSortChange}
           >
-            {this.sortCriteriaList.map((sortCriteria) => (
+            {MOVIE_SORT_CRITERIA.map((sortCriteria) => (
               <option key={sortCriteria.code} value={sortCriteria.code}>
                 {sortCriteria.label}
               </option>
             ))}
           </select>
         </div>
         <div className="container-side-margin text-white">
-          <strong>{this.state.filteredMovies.length}</strong> movies found.
+          <strong>{this.props.movies.length}</strong> movies found.
         </div>
         <div className="flex-grid movie-grid">
-          {this.state.filteredMovies.map((movie, i) => (
+          {this.props.movies.map((movie, i) => (
```

Comment:
as i see you useed more that one time this.props.movies
pls get movies destructively from props, and use everywhere only movies variable